docs/gallery/static/binaural_synthesis.ipynb

```diff
@@ -0,0 +1,469 @@
+      "text/plain": [
+       "<Figure size 480x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sources.show()\n",
+    "\n",
+    "ax = plt.gca()\n",
+    "ax.set_box_aspect([\n",
+    "    np.ptp(sources.x),\n",
+    "    np.ptp(sources.y),\n",
+    "    0.1,\n",
+    "])\n",
+    "ax.view_init(90, -90, 0)\n",
+    "ax.view_init(90, -90, 0)\n",
+    "ax.set_zticklabels([])\n",
+    "ax.set_zlabel('')\n",
+    "ax.set_in_layout(False)"
```

Comment:
What do you think of shortening it to
```suggestion
    "%matplotlib ipympl",
    "sources.show()\n",
```
I think with only the horizontal plane, the plot looks ok. When downloaded, one got get the 3D view. I wonder if a lot of matplotlib related code here is kind of distracting for newbies.